### ISSUE TITLE
Crash when placing Dead Counter

### ISSUE THREAD

**Issue by Grabsky** [Samlegamer/Macaw-s-Furnitures-Compat#10]
Hopefully this is the right place to report it. Please let me know if you need anything else.

**Minecraft**: `1.21.1`
**Fabric Loader**: `0.16.5`
**Fabric API**: `0.104.0+1.21.1`

**Macaw's Furniture**: `3.3.0`
**Mod**: `1.21.1-1.0`

**Item ID**: `mcwfurnituresbop:dead_counter`

```
[22:53:36] [Render thread/ERROR]: Unreported exception thrown!
java.lang.IllegalStateException: Invalid block entity mcwfurnitures:box_block // net.kikoz.mcwfurnitures.storage.FurniturekEntity state at class_2338{x=8, y=64, z=-42}, got Block{mcwfurnituresbop:dead_counter}[facing=west]
	at net.minecraft.class_2586.method_61175(class_2586.java:57) ~[client-intermediary.jar:?]
	at net.minecraft.class_2586.<init>(class_2586.java:51) ~[client-intermediary.jar:?]
	at net.kikoz.mcwfurnitures.storage.FurniturekEntity.<init>(FurniturekEntity.java:22) ~[mcw-furniture-3.3.0-mc1.21.1fabric.jar:?]
	at net.kikoz.mcwfurnitures.objects.FurnitureObjectNonFaceable.method_10123(FurnitureObjectNonFaceable.java:82) ~[mcw-furniture-3.3.0-mc1.21.1fabric.jar:?]
	at net.minecraft.class_2818.redirect$cej000$lithium$createBlockEntityWithCachedStateFix(class_2818.java:3256) ~[client-intermediary.jar:?]
	at net.minecraft.class_2818.method_12010(class_2818.java:286) ~[client-intermediary.jar:?]
	at net.minecraft.class_1937.method_30092(class_1937.java:232) ~[client-intermediary.jar:?]
	at net.minecraft.class_638.method_30092(class_638.java:160) ~[client-intermediary.jar:?]
	at net.minecraft.class_1937.method_8652(class_1937.java:217) ~[client-intermediary.jar:?]
	at net.minecraft.class_1747.method_7708(class_1747.java:153) ~[client-intermediary.jar:?]
	at net.minecraft.class_1747.method_7712(class_1747.java:76) ~[client-intermediary.jar:?]
	at net.minecraft.class_1747.method_7884(class_1747.java:46) ~[client-intermediary.jar:?]
	at net.minecraft.class_1799.method_7981(class_1799.java:373) ~[client-intermediary.jar:?]
	at net.minecraft.class_636.method_41934(class_636.java:363) ~[client-intermediary.jar:?]
	at net.minecraft.class_636.method_41933(class_636.java:318) ~[client-intermediary.jar:?]
	at net.minecraft.class_636.method_41931(class_636.java:279) ~[client-intermediary.jar:?]
	at net.minecraft.class_636.method_2896(class_636.java:317) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1583(class_310.java:1852) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1508(class_310.java:2111) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java:1938) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1246) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:882) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:256) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.16.5.jar:?]
```

**Comment by Grabsky** [comment]
Looks like more Biomes O' Plenty counters added by this mod are affected, not only Dead one.

**Comment by Samlegamer** [comment]
fixed in 1.21.1-1.1 https://legacy.curseforge.com/minecraft/mc-mods/macaws-furnitures-biomes-o-plenty/files/5753465
